judging/scripts/prescribe/dummy_prescriptor_validation.py

```diff
@@ -0,0 +1 @@
+print('Inside validation')
```

Comment:
This is the dummied up validation. See my comments in Slack about this.

Reply:
I can take care of it